test/bigtest/interactors/inventory.js

```diff
@@ -37,6 +37,8 @@ import {
   exportInstancesJSONBtnIsVisible = isVisible('#dropdown-clickable-export-json');
   isExportInstancesJSONBtnDisabled = property('#dropdown-clickable-export-json', 'disabled');
   isExportInstancesJSONIconPresent = isPresent('#dropdown-clickable-export-json [class*=icon-download]');
+  showSelectedRecordsBtn = new Interactor('#dropdown-clickable-show-selected-records');
```

Comment:
Let's use `ButtonInteractor` instead so `isShowSelectedRecordsBtnDisabled` won't be needed. I am doing the same in my PR.

Reply:
By the way, my [addition](https://github.com/folio-org/stripes-components/pull/1469) in the stripes-components to have `isDisabled` to be a part of the interactor won't propagate there as ui-inventory depends on stripes-components: ^8.0.0 but `master` has 9.0.0. For the disabled use `showSelectedRecordsBtn .$root.disabled` for now. We will change to `showSelectedRecordsBtn .isDisabled` after the release of `ui-inventory`.